Apsitvarkom.DataAccess/PollutedLocationDTOFileRepository.cs

```diff
@@ -75,9 +75,9 @@ public static PollutedLocationDTOFileRepository FromContent(IMapper mapper, stri
     /// <summary>Checks if the file path points to a file with .json extension.</summary>
     /// <param name="path">Path being checked.</param>
     /// <returns>Path on success, <see cref="FormatException"/> otherwise.</returns>
-    private static string CheckIfFileIsJson(string path)
+    private static string ValidateIfFilePathIsvalid(string path)
     {   
-        const string regexPattern = @"(\.json)$";
+        const string regexPattern = @"([a-zA-Z0-9_\\/.\-\(\):])+(.json)$";
```

Comment:
you can use \w to simplify the a-zA-Z0-9 part.

you also should escape the dot, as with your pattern asdasajson passes, as . is a symbol for any character.